src/constants.ts

```diff
@@ -30,12 +30,14 @@ export const getControlTreatmentsWithConfig = (featureFlagNames: unknown): Split
 };
 
 // Warning and error messages
-export const WARN_SF_CONFIG_AND_FACTORY: string = '[WARN] Both a config and factory props were provided to SplitFactory. Config prop will be ignored.';
+export const WARN_SF_CONFIG_AND_FACTORY: string = '[WARN]  Both a config and factory props were provided to SplitFactory. Config prop will be ignored.';
```

Comment:
Are these extra spaces intentional?

Reply:
Yes. It is intended so that log messages are aligned:
```
[ERROR] XXX
[WARN]  XXX
```